scripts/install_go_protobuf.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+GIT_TAG=v1.20.0
```

Comment:
Wait actually, why doesn't the latest version of protobuf work? On my local machine with modules it works for me.

Reply:
Thanks -- it broke something for me in environment -- I think that this was unnecessary once I'd added the go mod support.